packages/services/server/.env.template

```diff
@@ -33,6 +33,9 @@ ENCRYPTION_SECRET="97e4094d2463e71a981913cca4e56788"
 FEEDBACK_SLACK_TOKEN=""
 FEEDBACK_SLACK_CHANNEL="#hive"
 
+# Internal calls
+INTERNAL_ACCESS_SIGNATURE="123456"
```

Comment:
Nit: `AUTH_LEGACY_AUTH0_INTERNAL_API_KEY` could now also use this instead (just sayin)

Reply:
`INTERNAL_ACCESS_SIGNATURE` is something I'm going to remove, since the API calls are server-side.